lib/buildEmbed.js

```diff
@@ -16,24 +17,39 @@ module.exports = function(tweet, options, index) {
    * Default embed, which does NOT make a network oEmbed call to save the Tweet text as static HTML.
    * @param {Object} tweet
    * @param {Object} options
-   * @param {Int} index
-   *
+   * @param {Number} index
    */
 	function defaultEmbed(tweet, options, index) {
+		let quoteOptions = '';
+
+		if (options.theme === 'dark') {
+			quoteOptions += 'data-theme="dark"'
```

Comment:
```suggestion
			quoteOptions += ' data-theme="dark"'
```

Since you `trim` the custom attributes string anyway before appending, I'm inclined to add a leading space on all the individual options so the pattern is consistent and we can be agnostic about what order they go in (especially if we add more options in future).

Reply:
Great suggestion. I completely agree.